extensions/wikia/Places/PlacesController.class.php

```diff
@@ -132,6 +146,16 @@ public function getMapSnippet() {
 	 * Create a new place based on geo data provided and store it in the database
 	 */
 	public function saveNewPlaceToArticle(){
+		// SUS-1638: verify edit token
+		try {
+			$this->checkWriteRequest();
+		} catch ( BadRequestException $badRequestException ) {
+			$this->response->setValues( [
+				'success' => false,
+				'error' => wfMessage( 'sessionfailure' )->escaped()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)